.flowconfig

```diff
@@ -0,0 +1,51 @@
+[ignore]
+; We fork some components by platform
+.*/*[.]android.js
+
+; Ignore unexpected extra "@providesModule"
+.*/node_modules/.*/node_modules/fbjs/.*
+
+; Ignore duplicate module providers
+; For RN Apps installed via npm, "Libraries" folder is inside
+; "node_modules/react-native" but in the source repo it is in the root
+.*/Libraries/react-native/React.js
+.*/Libraries/react-native/ReactNative.js
+
+; Ignore any possible duplicates from within our Example folder
+.*/Example/node_modules/.*
```

Comment:
This is the only real custom magic here. We ignore `Example`'s `node_modules`, since it has duplicates with our `node_modules`